lang/c++/include/avro/Reader.hh

```diff
@@ -176,15 +176,15 @@ private:
         return encoded;
     }
 
-    int64_t readSize() {
+    size_t readSize() {
```

Comment:
This one looks a bit scary to me. It introduces a breaking change by changing the sign-ness of the return value. If not careful, it can be dangerously misrepresented on the user side.

Reply:
Both `readSize` and `readCount` are private, I don't believe this change is publicly visible (aside from projects with wConversion no longer having to suppress warnings when including avro, which is what motivated me to make this change)

Reply:
Oh sorry, I overlooked it.